stock_average_daily_sale/tests/common.py

```diff
@@ -50,20 +50,34 @@ def setUpClass(cls):
 
     @classmethod
     def _create_inventory(cls):
-        cls.inventory_obj.create(
+        inventory = cls.env["stock.inventory"].create(
```

Comment:
Not sure why we switched from `stock.quant` to `stock.inventory` to put goods in stock, we can also use quant model to do that in v14. E.g:
```python
        cls.env["stock.quant"]._update_available_quantity(
            product,
            location,
            quantity,
            package_id=package,
            lot_id=lot,
            in_date=in_date,
        )
```
Or if we want to state the current qty in stock (not adding new ones), we can use this helper method:
```python
    @classmethod
    def _update_qty_in_location(
        cls, location, product, quantity, package=None, lot=None, in_date=None
    ):
        quants = cls.env["stock.quant"]._gather(
            product, location, lot_id=lot, package_id=package, strict=True
        )
        # this method adds the quantity to the current quantity, so remove it
        quantity -= sum(quants.mapped("quantity"))
        cls.env["stock.quant"]._update_available_quantity(
            product,
            location,
            quantity,
            package_id=package,
            lot_id=lot,
            in_date=in_date,
        )
```

But probably not blocking for a backport.